src/shell.js

```diff
@@ -43,20 +43,11 @@ var Module = typeof {{{ EXPORT_NAME }}} !== 'undefined' ? {{{ EXPORT_NAME }}} :
 // modularize which creates a Promise for when the module is ready.
 #include "polyfill/promise.js"
 #endif
-#endif
 
 // See https://caniuse.com/mdn-javascript_builtins_object_assign
 #if MIN_CHROME_VERSION < 45 || MIN_EDGE_VERSION < 12 || MIN_FIREFOX_VERSION < 34 || MIN_IE_VERSION != TARGET_NOT_SUPPORTED || MIN_SAFARI_VERSION < 90000
-function objAssign(target, source) {
-  for (var key in source) {
-    if (source.hasOwnProperty(key)) {
-      target[key] = source[key];
-    }
-  }
-  return target;
-}
-#else
-var objAssign = Object.assign;
+#include "polyfill/objassign.js"
```

Comment:
Should this not be guarded behind `#if POLYFILL` so that flag is respected?

Reply:
It is yes, the #endif for the POLYFILL ifdef was moved from line 46 down to line 50.  

I'll add a comment to make it clear

Reply:
Ah, thanks, now I see it...